### ISSUE TITLE
<04/09/23><NYU><Android><Talkback> Background icon is missing in cancel ride popup<Parent_ID_1980>

### ISSUE THREAD

**Issue by Vinaykm999** [nammayatri/nammayatri#2906]
#1980 
[Background icon is missing in cancel ride popup ](https://drive.google.com/file/d/1V2E07grvTWBFdzwYwf5qcSbd9iv7O11W/view?usp=sharing)

**Steps**:
1. Request for a ride
2. Assign a ride
3. Click on cancel ride
4. verify popup screen 

**Actual** : background icon is missing

**Comment by MercyQueen** [comment]
Fixed.

**Comment by Vinaykm999** [comment]
Working fine
